docs/pascal_voc_user_manual.md

```diff
@@ -0,0 +1,298 @@
+
+It is also possible to import specific tasks
+of Pascal VOC dataset instead of the whole dataset,
+for example:
+
+``` bash
+datum add path -f voc_detection <path/to/dataset/ImageSets/Main/train.txt>
+```
+
+Datumaro supports the following Pascal VOC tasks:
+- Image classification (`voc_classification`)
+- Object detection (`voc_detection`)
+- Action classification (`voc_action`)
+- Class and instance segmentation (`voc_segmentation`)
+- Person layout detection (`voc_layout`)
+
+To make sure that the selected dataset has been added to the project, you can run
+`datum info`, which will display the project and dataset information.
+
+## Export to other formats
+
+Datumaro can convert Pascal VOC dataset into any other format
+[Datumaro supports](../docs/user_manual.md#supported-formats).
+
+Such conversion will only be successful if the output
+format can represent the type of dataset you want to convert,
+e.g. image classification annotations can be
+saved in `ImageNet` format, but no as `COCO keypoints`.
+
+There are few ways to convert Pascal VOC dataset to other dataset format:
+
+``` bash
+datum project import -f voc -i <path/to/voc>
+datum export -f coco -o <path/to/output/dir>
+# or
+datum convert -if voc -i <path/to/voc> -f coco -o <path/to/output/dir>
+```
+
+Also it is possible using filters for converting, check
+[user manual](../docs/user_manual.md#filter-project)
+for information about filtering:
+
+``` bash
+datum convert -if voc -i <path/to/voc> \
+    -f yolo -o <path/to/output/dir> \
+    --filter-mode FILTER_MODE \
+    --filter '<xpath filter expression>'
+```
+
+Some formats provide extra options for conversion.
+To get information about them, run
+`datum export -f <FORMAT> -- -h`
+These options are passed after double dash (`--`) in the command line.
+For example, the `voc_segmentation` format has an extra argument
+`--label_map`, which provides an option to load a custom color map for
+segmentation masks:
+
+``` bash
+datum export -f voc_segmentation -- --label-map mycolormap.txt
+
+# mycolormap.txt [label : color_rgb : parts : actions]:
+# cat:0,0,255::
+# person:255,0,0:head:
+```
+
+Also you can use original voc format of label map, for example:
+
+``` bash
+datum export -f voc_layout -- --label-map voc
+```
+
+## Export to Pascal VOC
+
+There are few ways to convert dataset to Pascal VOC format:
+
+``` bash
+# export dataset into Pascal VOC format (classification) from existing project
+datum export -p <path/to/project> -f voc -o <path/to/export/dir> -- --tasks classification
+
+# converting to Pascal VOC format from other format
+datum convert -if imagenet -i <path/to/imagenet/dataset> \
+    -f voc -o <path/to/export/dir> \
+    -- --label_map voc --save-images
+```
+
+Argument `--tasks` allow to specify tasks for export dataset,
+by default Datumaro uses all tasks.
+Argument   `--label_map` allow to define user label map, for example
```

Comment:
Add descriptions for all the export parameters. Add information how to use colormap to change colors in an existing dataset. Add information how colormap should look like to import a dataset in the grayscale format.